modules/nf-core/software/functions.nf

```diff
@@ -41,17 +41,19 @@ def saveFiles(Map args) {
     if (!args.filename.endsWith('.version.txt')) {
         def ioptions = initOptions(args.options)
         def path_list = [ ioptions.publish_dir ?: args.publish_dir ]
-        if (ioptions.publish_by_id) path_list.add(args.publish_id)
+        if (ioptions.publish_by_id) {
+            path_list.add(args.publish_id)
+        }
```

Comment:
Since the functions file is now within each tool/module folder, is this one still needed?